rfcs/text/0017_url_service.md

```diff
@@ -324,32 +437,59 @@ interface IShortUrlClient {
    * 
    * @param slug The slug (ID) of the short URL.
    */
-  get(slug: string): Promise<{ slug: string; shortUrl: object; url: URL }>;
+  get(slug: string): Promise<ShortUrl>;
 
   /**
    * Same as `get()` but it also increments the "view" counter and the
    * "last view" timestamp of this short URL.
    * 
    * @param slug The slug (ID) of the short URL.
    */
-  resolve(slug: string): Promise<{ slug: string; shortUrl: object; url: URL }>;
+  resolve(slug: string): Promise<ShortUrl>;
 }
 ```
 
-Note, that in the new service to generate a short URL the developer will have to
-use a URL generator.
+Note, that in this new service to create a short URL the developer will have to
```

Comment:
should we even have this ? seems we can achieve everything thru locators:
`plugins.discover.locator.createShortUrl(...)` if we know the target locator at compile time or
`plugins.share.locators.get(locatorId).createShortUrl(state)` when we don't know locator at compile time

Reply:
(the create method)

Reply:
I've added it here as it felt alien to have a CRUD API without the "create" method. But it is not strictly necessary, removed in the RFC.